app/validators/texts_equal_validator.rb

```diff
@@ -0,0 +1,9 @@
+class TextsEqualValidator < ActiveModel::Validator
+  def validate(record)
+    if record.original_text? &&
+      record.translated_text? &&
+      record.original_text.downcase.strip.split(' ') == record.translated_text.downcase.strip.split(' ')
```

Comment:
Line is too long. [104/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.